src/pages/milvus-demos/index.jsx

```diff
@@ -53,6 +54,12 @@ const MilvusDemos = ({ data, pageContext }) => {
     .childI18N.v2;
   const { locale } = pageContext;
 
+  const { search } = globalHistory.location;
+  const source =
+    search === '?utm_source=Reddit&utm_medium=cpc&utm_campaign=psocial'
```

Comment:
not use === in case market change the value, ['utm_source','utm_medium','utm_campaign'].every(v=>search.includes(v))